### PR TITLE
DOCSP-14265 document Include Driver Syntax for all languages

### PR DESCRIPTION
**Pull request by kanchana-mongodb**
- [JIRA](https://jira.mongodb.org/browse/DOCS-14265)
- [Stage](https://docs-mongodbcom-staging.corp.mongodb.com/compass/docsworker-xlarge/DOCSP-14265/export-pipeline-to-language/#optional-include-driver-syntax-if-desired)

### CODE REVIEW

File: source/includes/steps-export-pipeline-to-language.yaml [hunk 1]
```diff
@@ -42,20 +42,33 @@ content: |
...
+---
+title: Click :icon-fa5:`copy`.
 level: 4
 ref: click-copy
 content: |
-  Click the :guilabel:`Copy` button at the top-right of the pipeline
-  to copy the pipeline for the selected language to your clipboard.
-  You can now integrate and execute your created pipeline into your
-  application.
+  Click :icon-fa5:`copy` at the top-right of the pipeline to copy the
```
Comment (andf-mongodb) [inline source/includes/steps-export-pipeline-to-language.yaml:69]: [ out-of-scope ] While this icon change seeks to be helpful, it is B&W and mirrored compared to the screenshot icon (which is green). I know this is likely a site-wide design choice (B&W for readability), but it did cause me to search for matching icon in screenshot for a good minute before realize color mismatch.

File: source/includes/steps-export-pipeline-to-language.yaml [hunk 2]
```diff
@@ -42,20 +42,33 @@ content: |
...
 ---
-title: Click :guilabel:`Copy`.
+title: Include driver syntax, if desired.
+optional: true
+level: 4
+ref: include-driver-syntax
+content: |
+  Check the :guilabel:`Include Driver Syntax` option to include 
+  application code for the language selected. If you include driver 
+  syntax, the copyable code reflects :ref:`project
```
Comment (andf-mongodb) [inline source/includes/steps-export-pipeline-to-language.yaml:59]: [nit] perhaps:
- "reflects the configured X, Y, and Z options",  or 
- "reflects your configured X, Y, and Z options"

Would it be useful to indicate where I configured these options? i.e.

- "reflects the X, Y, and Z options as configured in <Fantastic_Admin_Panel_5>"

**Review by andf-mongodb — APPROVED** [review]
LGTM mod tiny nits.

File: source/includes/steps-export-pipeline-to-language.yaml (hunk 1 above)
Comment (kanchana-mongodb) [reply source/includes/steps-export-pipeline-to-language.yaml:69]: see [PR](https://github.com/mongodb/docs-tools/pull/488) for the icon. Once the PR is merged, I'll fix the icon.

Comment (andf-mongodb) [reply source/includes/steps-export-pipeline-to-language.yaml:69]: Nice, looks good!

**Pull request merged**